accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -501,8 +504,9 @@ was marked `late`.  Note that this is a change from pre-NNBD semantics in that:
 
 ## Core library changes
 
-Calling the `.length` setter on a `List` of non-nullable element type with an
-argument greater than the current length of the list is a runtime error.
+Calling the `.length` setter on a `List` with element type `E` with an argument
```

Comment:
No, potentially non-nullable is defined via the appropriate syntactic condition and is still correct here.

Reply:
So this one is "unless E is definitely nullable".

(Still unsound because of `List<int?> l = <int>[1]; l.length = 2;` will fail at run-time. Nothing new here, covariant generics are unsound when you mutate the object).